### PR TITLE
[CryptofeedWebsocketConnector] Fix candle subscription

### PR DESCRIPTION
**Pull request by Herklos**

### CODE REVIEW

File: octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py [hunk 1]
```diff
@@ -388,7 +388,7 @@ def _subscribe_candle_feed(self):
                                  symbols=self.filtered_pairs,
                                  log_message_on_error=True,
                                  channels=self.CANDLE_CHANNELS,
-                                 callbacks=self.callbacks)
+                                 callbacks={cryptofeed_constants.CANDLES: self.candle_callback})
```
Comment (GuillaumeDSM) [inline octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py:391]: LOL

**Review by GuillaumeDSM — APPROVED** [review]

File: octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py [hunk 2]
```diff
@@ -423,6 +423,8 @@ def _subscribe_watched_pairs_feed(self):
                                  log_message_on_error=True,
                                  channels=channels,
                                  callbacks=self.callbacks)
+            for channel in channels:
+                self.logger.debug(f"Subscribed to {channel} for {', '.join(self.watched_pairs)}")
```
Comment (GuillaumeDSM) [inline octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py:427]: 👍

**Review by GuillaumeDSM — APPROVED** [review]

**Comment by GuillaumeDSM** [comment]
we attack a version commit ?

**Comment by Herklos** [comment]
> we attach a version commit ?

Yes

**Review by GuillaumeDSM — APPROVED** [review]

**Pull request merged**